WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -737,6 +741,10 @@ function (ev) {
     var arr = this.widget.getSelection();
     if(arr && arr.length) {
         arr.sort();
+        // When retrieving data from view make sure to html decode it
+        arr = arr.map(function(val) {
```

Comment:
Can we have a common utility somewhere in zm-ajax library as I see a pattern where similar functions are being used again & again. ?